content/en/about/contributing/shortcodes/_index.md

```diff
@@ -30,8 +30,7 @@ The following code:
 
 Will render as:
 
-{{% pageinfo %}} This is a
-placeholder. {{% /pageinfo %}}
+{{% pageinfo %}} This is a placeholder. {{% /pageinfo %}}
```

Comment:
This wrapping changes are a bit distracting. I understand they can happen but it's just adding noise

Reply:
It happened as the file wasn't formatted with prettier.